lib/src/authenticate.tsx

```diff
@@ -156,21 +174,18 @@ const AuthProvider: FunctionComponent<PropsWithChildren<AuthProviderPropsInterfa
                     || authParams?.authorizationCode
                     || url.searchParams.get("error") )
                 {
-                    await signIn(
-                        { callOnlyOnRedirect: true }, 
-                        authParams?.authorizationCode, 
-                        authParams?.sessionState,
-                        authParams?.state
-                        )
-                        .then(() => {
-                            // TODO: Add logs when a logger is available.
-                            // Tracked here https://github.com/asgardeo/asgardeo-auth-js-sdk/issues/151.
-                        })
-                        .catch((error) => {
-                            // TODO: Add logs when a logger is available.
-                            // Tracked here https://github.com/asgardeo/asgardeo-auth-js-sdk/issues/151.
-                            throw error;
-                        });
+                    try{
+                        await signIn(
+                            { callOnlyOnRedirect: true }, 
+                            authParams?.authorizationCode, 
+                            authParams?.sessionState,
+                            authParams?.state
+                        );
+                    } catch(error) {
+                        if(error?.code === "JS-CRYPTO_UTILS-IVIT-IV02") {
```

Comment:
Better if these error codes are exported from `auth-js` SDK. We can track an issue for this.
```suggestion
                        if(error?.code === "JS-CRYPTO_UTILS-IVIT-IV02") {
```